CONSTANTS.py

```diff
@@ -0,0 +1,7 @@
+""" CONSTANTS module """
```

Comment:
No need for all caps.
```suggestion
""" client constants """
```

Reply:
This applies everywhere.